gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Wed Mar 08 18:16:17 CET 2017
+#Thu Mar 21 20:32:52 CET 2019
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.4-all.zip
```

Comment:
I think there was some issue (in the past?) preventing paintroid from upgrading to gradle > 4.4 - I think it was some build dependency. I will see if I find out what issues we had. Might just have been our bintray upload plugin, which won't be an issue for catroid.